utility/src/main/java/org/oppia/util/parser/SvgSoftwareLayerSetter.kt

```diff
@@ -0,0 +1,33 @@
+package org.oppia.util.parser
+
+import android.graphics.drawable.PictureDrawable
+import com.bumptech.glide.load.DataSource
+import com.bumptech.glide.load.engine.GlideException
+import com.bumptech.glide.request.RequestListener
+import com.bumptech.glide.request.target.Target
+
+/**
+ * Listener which updates the {@link ImageView} to be software rendered, because {@link
+ * com.caverock.androidsvg.SVG SVG}/{@link android.graphics.Picture Picture} can't render on a
```

Comment:
Is this a limitation in the SVG library? From the Android documentation ``Picture``s can be rendered on hardware-backed surfaces on SDK 23+. It seems worthwhile to allow that optimization for SDKs that support it. Or, did misunderstand the documentation?

https://developer.android.com/reference/android/graphics/Picture

Reply:
According to the PictureDrawable docs, Canvas doesn't support rendering Pictures/SVGs on hardware canvases until Marshmallow.

Reply:
Can we mention in the docs that it's fine to use a hardware canvas for SDK versions 23+? That's more of what I was getting at--if it's actually possible, we should call out that it's allowed since we may want to do this in the future (e.g. check if we're running on SDK 23+ and, if so, use a hardware-accelerated layout to improve image rendering performance).

Reply:
Added in doc.